request.go

```diff
@@ -1268,7 +1267,7 @@ func NewUpsertRequest(space interface{}) *UpsertRequest {
 	req.rtype = iproto.IPROTO_UPSERT
 	req.setSpace(space)
 	req.tuple = []interface{}{}
-	req.ops = []interface{}{}
+	req.ops = NewOperations()
```

Comment:
Let's not create an empty object here, this leads to unnecessary allocation to the request.

It's better to check `req.ops == nil` during encoding and use []interface{}{} instead if so.

Reply:
Updated.